pyaedt/application/AnalsyisEmit.py

```diff
@@ -8,34 +8,31 @@
 
 
 class FieldAnalysisEmit(Design):
-    """**AEDT Emit Analysis**
+    """FieldAnaysisEmit  class.
 
-    Class for Emit Analysis Setup (Emit)
-
-    It is automatically initialized by Application call (like HFSS,
-    Q3D...). Refer to Application function for inputs definition
+    The class is for setting up an EMIT analysis in AEDT.
+    It is automatically initialized by an application call (like HFSS,
+    Q3D, and ...). Refer to the Application function for input definitions.
 
     Parameters
     ----------
 
-    Returns
-    -------
-
+    
     """
     @property
     def solution_type(self):
-        """ """
+        """Solution type."""
         return self._solution_type
 
 
     @solution_type.setter
     def solution_type(self, soltype):
-        """Solution Type
+        """Solution type.
```

Comment:
Documentation strings are not necessary for setters.  Move much of this to the `solution_type` property.